charts/aws-ebs-csi-driver/templates/snapshot-controller.yaml

```diff
@@ -57,12 +57,12 @@ spec:
             {{- include "aws-ebs-csi-driver.http-proxy" . | nindent 12 }}
           {{- end }}
           args:
-            - --v=5
+            - --v={{ .Values.snapshotController.logLevel }}
```

Comment:
You should remove this since the snapshot controller will soon no longer be installed via this chart